core/src/epicli/data/common/ansible/playbooks/zookeeper.yml

```diff
@@ -9,4 +9,5 @@
   become: true
   become_method: sudo
   roles:
-    - zookeeper
+    - role: zookeeper
+      when: ansible_architecture != "aarch64"
```

Comment:
\n

Reply:
We want to have kafka on ARM but without zookeeper?

Reply:
zookeeper is needed, good point, Ill revert it back